test_unstructured_ingest/expected-structured-output/sharepoint/Shared Documents/fake-text.json

```diff
@@ -0,0 +1,254 @@
+[
+  {
+    "type": "NarrativeText",
+    "element_id": "1df8eeb8be847c3a1a7411e3be3e0396",
+    "metadata": {
+      "data_source": {
+        "record_locator": {
+          "CheckInComment": "",
+          "CheckOutType": 2,
+          "ContentTag": "{880F80CA-CEBF-48D0-B639-AEB671B3C431},1,1",
+          "CustomizedPageStatus": 0,
+          "ETag": "\"{880F80CA-CEBF-48D0-B639-AEB671B3C431},1\"",
+          "Exists": true,
+          "ExistsAllowThrowForPolicyFailures": true,
+          "IrmEnabled": false,
+          "Length": "169",
+          "Level": 1,
+          "LinkingUri": null,
+          "LinkingUrl": "",
+          "MajorVersion": 1,
+          "MinorVersion": 0,
+          "Name": "fake-text.txt",
+          "ServerRelativeUrl": "/Shared Documents/fake-text.txt",
+          "TimeCreated": "2023-06-16T05:04:55Z",
+          "TimeLastModified": "2023-06-16T05:04:55Z",
+          "Title": null,
+          "UIVersion": 512,
+          "UIVersionLabel": "1.0",
+          "UniqueId": "880f80ca-cebf-48d0-b639-aeb671b3c431",
+          "ServerRelativePath": {
+            "DecodedUrl": "/Shared Documents/fake-text.txt"
+          }
+        },
+        "date_created": "2023-06-16T05:04:55Z",
+        "date_modified": "2023-06-16T05:04:55Z",
+        "date_processed": "2023-07-31T07:56:48.088321"
+      },
+      "filename": "fake-text.txt",
+      "file_directory": "/Users/rvztz/.cache/unstructured/ingest/95f260da5f/Shared Documents",
+      "filetype": "text/plain"
+    },
+    "text": "This is a test document to use for unit tests."
+  },
+  {
+    "type": "Address",
+    "element_id": "a9d4657034aa3fdb5177f1325e912362",
+    "metadata": {
+      "data_source": {
+        "record_locator": {
+          "CheckInComment": "",
+          "CheckOutType": 2,
+          "ContentTag": "{880F80CA-CEBF-48D0-B639-AEB671B3C431},1,1",
+          "CustomizedPageStatus": 0,
+          "ETag": "\"{880F80CA-CEBF-48D0-B639-AEB671B3C431},1\"",
+          "Exists": true,
+          "ExistsAllowThrowForPolicyFailures": true,
+          "IrmEnabled": false,
+          "Length": "169",
+          "Level": 1,
+          "LinkingUri": null,
+          "LinkingUrl": "",
+          "MajorVersion": 1,
+          "MinorVersion": 0,
+          "Name": "fake-text.txt",
+          "ServerRelativeUrl": "/Shared Documents/fake-text.txt",
+          "TimeCreated": "2023-06-16T05:04:55Z",
+          "TimeLastModified": "2023-06-16T05:04:55Z",
+          "Title": null,
+          "UIVersion": 512,
+          "UIVersionLabel": "1.0",
+          "UniqueId": "880f80ca-cebf-48d0-b639-aeb671b3c431",
+          "ServerRelativePath": {
+            "DecodedUrl": "/Shared Documents/fake-text.txt"
+          }
```

Comment:
why are there still expected results with all of these fields we decided to remove in it?

Reply:
looks like you addressed this.